crates/harness/src/test_runner.rs

```diff
@@ -8,8 +8,7 @@ use tracing::warn;
 use crate::util::random_unused_port;
 
 pub struct TestRunner {
-    /// The port that the balancer is listening on for client connections.
-    pub port: u16,
+    spawn_options: BalancerSpawnOptions,
```

Comment:
what does this do?

Reply:
BalancerSpawnOptions is used to hold the parameters used to start the balancer. They are held here to so they can be reused when restarting the balancer.